runtime/middlewares/logger/logger.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
This file does not implement new middleware interface, either refactor or delete.